{"code":"28P01","file":"auth.c","length":104,"level":"error","line":"324","message":"Unable to connect to database: password authentication failed for user \"postgres\"","name":"error","routine":"auth_failed","service":"user-service","severity":"FATAL","stack":"error: password authentication failed for user \"postgres\"\n    at C:\\Users\\39349\\OneDrive - Poste Italiane S.p.A\\Desktop OLD\\PROGETTI VISUAL STUDIO\\PERSONALE\\FANTACONTRATTI\\node_modules\\pg-pool\\index.js:45:11\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async connectDatabase (C:\\Users\\39349\\OneDrive - Poste Italiane S.p.A\\Desktop OLD\\PROGETTI VISUAL STUDIO\\PERSONALE\\FANTACONTRATTI\\services\\backend\\user-service\\src\\database\\connection.ts:20:20)\n    at async startServer (C:\\Users\\39349\\OneDrive - Poste Italiane S.p.A\\Desktop OLD\\PROGETTI VISUAL STUDIO\\PERSONALE\\FANTACONTRATTI\\services\\backend\\user-service\\src\\index.ts:62:5)","timestamp":"2025-09-22T14:28:43.767Z"}
{"code":"28P01","file":"auth.c","length":104,"level":"error","line":"324","message":"Unable to connect to database: password authentication failed for user \"postgres\"","name":"error","routine":"auth_failed","service":"user-service","severity":"FATAL","stack":"error: password authentication failed for user \"postgres\"\n    at C:\\Users\\39349\\OneDrive - Poste Italiane S.p.A\\Desktop OLD\\PROGETTI VISUAL STUDIO\\PERSONALE\\FANTACONTRATTI\\node_modules\\pg-pool\\index.js:45:11\n    at processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at connectDatabase (C:\\Users\\39349\\OneDrive - Poste Italiane S.p.A\\Desktop OLD\\PROGETTI VISUAL STUDIO\\PERSONALE\\FANTACONTRATTI\\services\\backend\\user-service\\src\\database\\connection.ts:20:20)\n    at Object.<anonymous> (C:\\Users\\39349\\OneDrive - Poste Italiane S.p.A\\Desktop OLD\\PROGETTI VISUAL STUDIO\\PERSONALE\\FANTACONTRATTI\\services\\backend\\user-service\\src\\__tests__\\integration.database.test.ts:18:5)","timestamp":"2025-09-22T16:32:04.031Z"}
{"code":"28P01","file":"auth.c","length":104,"level":"error","line":"324","message":"Unable to connect to database: password authentication failed for user \"postgres\"","name":"error","routine":"auth_failed","service":"user-service","severity":"FATAL","stack":"error: password authentication failed for user \"postgres\"\n    at C:\\Users\\39349\\OneDrive - Poste Italiane S.p.A\\Desktop OLD\\PROGETTI VISUAL STUDIO\\PERSONALE\\FANTACONTRATTI\\node_modules\\pg-pool\\index.js:45:11\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async connectDatabase (C:\\Users\\39349\\OneDrive - Poste Italiane S.p.A\\Desktop OLD\\PROGETTI VISUAL STUDIO\\PERSONALE\\FANTACONTRATTI\\services\\backend\\user-service\\dist\\database\\connection.js:22:24)\n    at async startServer (C:\\Users\\39349\\OneDrive - Poste Italiane S.p.A\\Desktop OLD\\PROGETTI VISUAL STUDIO\\PERSONALE\\FANTACONTRATTI\\services\\backend\\user-service\\dist\\index.js:51:9)","timestamp":"2025-09-23T14:19:10.613Z"}
{"code":"28P01","file":"auth.c","length":104,"level":"error","line":"324","message":"Unable to connect to database: password authentication failed for user \"postgres\"","name":"error","routine":"auth_failed","service":"user-service","severity":"FATAL","stack":"error: password authentication failed for user \"postgres\"\n    at C:\\Users\\39349\\OneDrive - Poste Italiane S.p.A\\Desktop OLD\\PROGETTI VISUAL STUDIO\\PERSONALE\\FANTACONTRATTI\\node_modules\\pg-pool\\index.js:45:11\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async connectDatabase (C:\\Users\\39349\\OneDrive - Poste Italiane S.p.A\\Desktop OLD\\PROGETTI VISUAL STUDIO\\PERSONALE\\FANTACONTRATTI\\services\\backend\\user-service\\dist\\database\\connection.js:22:24)\n    at async startServer (C:\\Users\\39349\\OneDrive - Poste Italiane S.p.A\\Desktop OLD\\PROGETTI VISUAL STUDIO\\PERSONALE\\FANTACONTRATTI\\services\\backend\\user-service\\dist\\index.js:51:9)","timestamp":"2025-09-23T14:48:54.639Z"}
{"code":"28P01","file":"auth.c","length":104,"level":"error","line":"324","message":"Unable to connect to database: password authentication failed for user \"postgres\"","name":"error","routine":"auth_failed","service":"user-service","severity":"FATAL","stack":"error: password authentication failed for user \"postgres\"\n    at C:\\Users\\39349\\OneDrive - Poste Italiane S.p.A\\Desktop OLD\\PROGETTI VISUAL STUDIO\\PERSONALE\\FANTACONTRATTI\\node_modules\\pg-pool\\index.js:45:11\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async connectDatabase (C:\\Users\\39349\\OneDrive - Poste Italiane S.p.A\\Desktop OLD\\PROGETTI VISUAL STUDIO\\PERSONALE\\FANTACONTRATTI\\services\\backend\\user-service\\dist\\database\\connection.js:22:24)\n    at async startServer (C:\\Users\\39349\\OneDrive - Poste Italiane S.p.A\\Desktop OLD\\PROGETTI VISUAL STUDIO\\PERSONALE\\FANTACONTRATTI\\services\\backend\\user-service\\dist\\index.js:51:9)","timestamp":"2025-09-23T16:40:33.801Z"}
{"code":"28P01","file":"auth.c","length":104,"level":"error","line":"324","message":"Unable to connect to database: password authentication failed for user \"postgres\"","name":"error","routine":"auth_failed","service":"user-service","severity":"FATAL","stack":"error: password authentication failed for user \"postgres\"\n    at C:\\Users\\39349\\OneDrive - Poste Italiane S.p.A\\Desktop OLD\\PROGETTI VISUAL STUDIO\\PERSONALE\\FANTACONTRATTI\\node_modules\\pg-pool\\index.js:45:11\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async connectDatabase (C:\\Users\\39349\\OneDrive - Poste Italiane S.p.A\\Desktop OLD\\PROGETTI VISUAL STUDIO\\PERSONALE\\FANTACONTRATTI\\services\\backend\\user-service\\src\\database\\connection.ts:20:20)\n    at async startServer (C:\\Users\\39349\\OneDrive - Poste Italiane S.p.A\\Desktop OLD\\PROGETTI VISUAL STUDIO\\PERSONALE\\FANTACONTRATTI\\services\\backend\\user-service\\src\\index.ts:62:5)","timestamp":"2025-09-24T06:13:19.454Z"}
{"code":"28P01","file":"auth.c","length":104,"level":"error","line":"324","message":"Unable to connect to database: password authentication failed for user \"postgres\"","name":"error","routine":"auth_failed","service":"user-service","severity":"FATAL","stack":"error: password authentication failed for user \"postgres\"\n    at C:\\Users\\39349\\OneDrive - Poste Italiane S.p.A\\Desktop OLD\\PROGETTI VISUAL STUDIO\\PERSONALE\\FANTACONTRATTI\\node_modules\\pg-pool\\index.js:45:11\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async connectDatabase (C:\\Users\\39349\\OneDrive - Poste Italiane S.p.A\\Desktop OLD\\PROGETTI VISUAL STUDIO\\PERSONALE\\FANTACONTRATTI\\services\\backend\\user-service\\src\\database\\connection.ts:20:20)\n    at async startServer (C:\\Users\\39349\\OneDrive - Poste Italiane S.p.A\\Desktop OLD\\PROGETTI VISUAL STUDIO\\PERSONALE\\FANTACONTRATTI\\services\\backend\\user-service\\src\\index.ts:62:5)","timestamp":"2025-09-24T06:22:53.564Z"}
{"code":"28P01","file":"auth.c","length":104,"level":"error","line":"324","message":"Unable to connect to database: password authentication failed for user \"postgres\"","name":"error","routine":"auth_failed","service":"user-service","severity":"FATAL","stack":"error: password authentication failed for user \"postgres\"\n    at C:\\Users\\39349\\OneDrive - Poste Italiane S.p.A\\Desktop OLD\\PROGETTI VISUAL STUDIO\\PERSONALE\\FANTACONTRATTI\\node_modules\\pg-pool\\index.js:45:11\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async connectDatabase (C:\\Users\\39349\\OneDrive - Poste Italiane S.p.A\\Desktop OLD\\PROGETTI VISUAL STUDIO\\PERSONALE\\FANTACONTRATTI\\services\\backend\\user-service\\src\\database\\connection.ts:20:20)\n    at async startServer (C:\\Users\\39349\\OneDrive - Poste Italiane S.p.A\\Desktop OLD\\PROGETTI VISUAL STUDIO\\PERSONALE\\FANTACONTRATTI\\services\\backend\\user-service\\src\\index.ts:62:5)","timestamp":"2025-09-24T06:27:29.316Z"}
{"code":"28P01","file":"auth.c","length":104,"level":"error","line":"324","message":"Unable to connect to database: password authentication failed for user \"postgres\"","name":"error","routine":"auth_failed","service":"user-service","severity":"FATAL","stack":"error: password authentication failed for user \"postgres\"\n    at C:\\Users\\39349\\OneDrive - Poste Italiane S.p.A\\Desktop OLD\\PROGETTI VISUAL STUDIO\\PERSONALE\\FANTACONTRATTI\\node_modules\\pg-pool\\index.js:45:11\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async connectDatabase (C:\\Users\\39349\\OneDrive - Poste Italiane S.p.A\\Desktop OLD\\PROGETTI VISUAL STUDIO\\PERSONALE\\FANTACONTRATTI\\services\\backend\\user-service\\src\\database\\connection.ts:20:20)\n    at async startServer (C:\\Users\\39349\\OneDrive - Poste Italiane S.p.A\\Desktop OLD\\PROGETTI VISUAL STUDIO\\PERSONALE\\FANTACONTRATTI\\services\\backend\\user-service\\src\\index.ts:62:5)","timestamp":"2025-09-24T06:32:04.940Z"}
{"code":"28P01","file":"auth.c","length":104,"level":"error","line":"324","message":"Unable to connect to database: password authentication failed for user \"postgres\"","name":"error","routine":"auth_failed","service":"user-service","severity":"FATAL","stack":"error: password authentication failed for user \"postgres\"\n    at C:\\Users\\39349\\OneDrive - Poste Italiane S.p.A\\Desktop OLD\\PROGETTI VISUAL STUDIO\\PERSONALE\\FANTACONTRATTI\\node_modules\\pg-pool\\index.js:45:11\n    at processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at connectDatabase (C:\\Users\\39349\\OneDrive - Poste Italiane S.p.A\\Desktop OLD\\PROGETTI VISUAL STUDIO\\PERSONALE\\FANTACONTRATTI\\services\\backend\\user-service\\src\\database\\connection.ts:20:20)\n    at Object.<anonymous> (C:\\Users\\39349\\OneDrive - Poste Italiane S.p.A\\Desktop OLD\\PROGETTI VISUAL STUDIO\\PERSONALE\\FANTACONTRATTI\\services\\backend\\user-service\\src\\__tests__\\database.integration.test.ts:18:7)","timestamp":"2025-09-24T09:16:40.101Z"}
{"code":"28P01","file":"auth.c","length":104,"level":"error","line":"324","message":"Unable to connect to database: password authentication failed for user \"postgres\"","name":"error","routine":"auth_failed","service":"user-service","severity":"FATAL","stack":"error: password authentication failed for user \"postgres\"\n    at C:\\Users\\39349\\OneDrive - Poste Italiane S.p.A\\Desktop OLD\\PROGETTI VISUAL STUDIO\\PERSONALE\\FANTACONTRATTI\\node_modules\\pg-pool\\index.js:45:11\n    at processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at connectDatabase (C:\\Users\\39349\\OneDrive - Poste Italiane S.p.A\\Desktop OLD\\PROGETTI VISUAL STUDIO\\PERSONALE\\FANTACONTRATTI\\services\\backend\\user-service\\src\\database\\connection.ts:20:20)\n    at Object.<anonymous> (C:\\Users\\39349\\OneDrive - Poste Italiane S.p.A\\Desktop OLD\\PROGETTI VISUAL STUDIO\\PERSONALE\\FANTACONTRATTI\\services\\backend\\user-service\\src\\__tests__\\database.integration.test.ts:28:7)","timestamp":"2025-09-24T09:16:40.162Z"}
{"code":"28P01","file":"auth.c","length":104,"level":"error","line":"324","message":"Unable to connect to database: password authentication failed for user \"postgres\"","name":"error","routine":"auth_failed","service":"user-service","severity":"FATAL","stack":"error: password authentication failed for user \"postgres\"\n    at C:\\Users\\39349\\OneDrive - Poste Italiane S.p.A\\Desktop OLD\\PROGETTI VISUAL STUDIO\\PERSONALE\\FANTACONTRATTI\\node_modules\\pg-pool\\index.js:45:11\n    at processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at connectDatabase (C:\\Users\\39349\\OneDrive - Poste Italiane S.p.A\\Desktop OLD\\PROGETTI VISUAL STUDIO\\PERSONALE\\FANTACONTRATTI\\services\\backend\\user-service\\src\\database\\connection.ts:20:20)\n    at Object.<anonymous> (C:\\Users\\39349\\OneDrive - Poste Italiane S.p.A\\Desktop OLD\\PROGETTI VISUAL STUDIO\\PERSONALE\\FANTACONTRATTI\\services\\backend\\user-service\\src\\__tests__\\database.integration.test.ts:43:7)","timestamp":"2025-09-24T09:16:40.311Z"}
{"code":"ENOTFOUND","errno":-3008,"hostname":"nonexistent_host","level":"error","message":"Unable to connect to database: getaddrinfo ENOTFOUND nonexistent_host","service":"user-service","stack":"Error: getaddrinfo ENOTFOUND nonexistent_host\n    at C:\\Users\\39349\\OneDrive - Poste Italiane S.p.A\\Desktop OLD\\PROGETTI VISUAL STUDIO\\PERSONALE\\FANTACONTRATTI\\node_modules\\pg-pool\\index.js:45:11\n    at processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at connectDatabase (C:\\Users\\39349\\OneDrive - Poste Italiane S.p.A\\Desktop OLD\\PROGETTI VISUAL STUDIO\\PERSONALE\\FANTACONTRATTI\\services\\backend\\user-service\\src\\database\\connection.ts:20:20)\n    at Object.<anonymous> (C:\\Users\\39349\\OneDrive - Poste Italiane S.p.A\\Desktop OLD\\PROGETTI VISUAL STUDIO\\PERSONALE\\FANTACONTRATTI\\services\\backend\\user-service\\src\\__tests__\\database.integration.test.ts:69:7)","syscall":"getaddrinfo","timestamp":"2025-09-24T09:16:41.618Z"}
{"code":"ENOTFOUND","errno":-3008,"hostname":"nonexistent_host","level":"error","message":"Unable to connect to database: getaddrinfo ENOTFOUND nonexistent_host","service":"user-service","stack":"Error: getaddrinfo ENOTFOUND nonexistent_host\n    at C:\\Users\\39349\\OneDrive - Poste Italiane S.p.A\\Desktop OLD\\PROGETTI VISUAL STUDIO\\PERSONALE\\FANTACONTRATTI\\node_modules\\pg-pool\\index.js:45:11\n    at processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at connectDatabase (C:\\Users\\39349\\OneDrive - Poste Italiane S.p.A\\Desktop OLD\\PROGETTI VISUAL STUDIO\\PERSONALE\\FANTACONTRATTI\\services\\backend\\user-service\\src\\database\\connection.ts:20:20)\n    at Object.<anonymous> (C:\\Users\\39349\\OneDrive - Poste Italiane S.p.A\\Desktop OLD\\PROGETTI VISUAL STUDIO\\PERSONALE\\FANTACONTRATTI\\services\\backend\\user-service\\src\\__tests__\\database.integration.test.ts:78:7)","syscall":"getaddrinfo","timestamp":"2025-09-24T09:16:42.767Z"}
{"code":"ENOTFOUND","errno":-3008,"hostname":"nonexistent_host","level":"error","message":"Unable to connect to database: getaddrinfo ENOTFOUND nonexistent_host","service":"user-service","stack":"Error: getaddrinfo ENOTFOUND nonexistent_host\n    at C:\\Users\\39349\\OneDrive - Poste Italiane S.p.A\\Desktop OLD\\PROGETTI VISUAL STUDIO\\PERSONALE\\FANTACONTRATTI\\node_modules\\pg-pool\\index.js:45:11\n    at processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at connectDatabase (C:\\Users\\39349\\OneDrive - Poste Italiane S.p.A\\Desktop OLD\\PROGETTI VISUAL STUDIO\\PERSONALE\\FANTACONTRATTI\\services\\backend\\user-service\\src\\database\\connection.ts:20:20)\n    at Object.<anonymous> (C:\\Users\\39349\\OneDrive - Poste Italiane S.p.A\\Desktop OLD\\PROGETTI VISUAL STUDIO\\PERSONALE\\FANTACONTRATTI\\services\\backend\\user-service\\src\\__tests__\\database.integration.test.ts:78:7)","syscall":"getaddrinfo","timestamp":"2025-09-24T09:16:43.919Z"}
{"code":"28P01","file":"auth.c","length":104,"level":"error","line":"324","message":"Unable to connect to database: password authentication failed for user \"postgres\"","name":"error","routine":"auth_failed","service":"user-service","severity":"FATAL","stack":"error: password authentication failed for user \"postgres\"\n    at C:\\Users\\39349\\OneDrive - Poste Italiane S.p.A\\Desktop OLD\\PROGETTI VISUAL STUDIO\\PERSONALE\\FANTACONTRATTI\\node_modules\\pg-pool\\index.js:45:11\n    at processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at connectDatabase (C:\\Users\\39349\\OneDrive - Poste Italiane S.p.A\\Desktop OLD\\PROGETTI VISUAL STUDIO\\PERSONALE\\FANTACONTRATTI\\services\\backend\\user-service\\src\\database\\connection.ts:39:24)\n    at Object.<anonymous> (C:\\Users\\39349\\OneDrive - Poste Italiane S.p.A\\Desktop OLD\\PROGETTI VISUAL STUDIO\\PERSONALE\\FANTACONTRATTI\\services\\backend\\user-service\\src\\__tests__\\database.integration.test.ts:18:7)","timestamp":"2025-09-24T09:20:01.734Z"}
{"code":"28P01","file":"auth.c","length":104,"level":"error","line":"324","message":"Unable to connect to database: password authentication failed for user \"postgres\"","name":"error","routine":"auth_failed","service":"user-service","severity":"FATAL","stack":"error: password authentication failed for user \"postgres\"\n    at C:\\Users\\39349\\OneDrive - Poste Italiane S.p.A\\Desktop OLD\\PROGETTI VISUAL STUDIO\\PERSONALE\\FANTACONTRATTI\\node_modules\\pg-pool\\index.js:45:11\n    at processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at connectDatabase (C:\\Users\\39349\\OneDrive - Poste Italiane S.p.A\\Desktop OLD\\PROGETTI VISUAL STUDIO\\PERSONALE\\FANTACONTRATTI\\services\\backend\\user-service\\src\\database\\connection.ts:39:24)\n    at Object.<anonymous> (C:\\Users\\39349\\OneDrive - Poste Italiane S.p.A\\Desktop OLD\\PROGETTI VISUAL STUDIO\\PERSONALE\\FANTACONTRATTI\\services\\backend\\user-service\\src\\__tests__\\database.integration.test.ts:28:7)","timestamp":"2025-09-24T09:20:04.100Z"}
{"code":"28P01","file":"auth.c","length":104,"level":"error","line":"324","message":"Unable to connect to database: password authentication failed for user \"postgres\"","name":"error","routine":"auth_failed","service":"user-service","severity":"FATAL","stack":"error: password authentication failed for user \"postgres\"\n    at C:\\Users\\39349\\OneDrive - Poste Italiane S.p.A\\Desktop OLD\\PROGETTI VISUAL STUDIO\\PERSONALE\\FANTACONTRATTI\\node_modules\\pg-pool\\index.js:45:11\n    at processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at connectDatabase (C:\\Users\\39349\\OneDrive - Poste Italiane S.p.A\\Desktop OLD\\PROGETTI VISUAL STUDIO\\PERSONALE\\FANTACONTRATTI\\services\\backend\\user-service\\src\\database\\connection.ts:39:24)\n    at Object.<anonymous> (C:\\Users\\39349\\OneDrive - Poste Italiane S.p.A\\Desktop OLD\\PROGETTI VISUAL STUDIO\\PERSONALE\\FANTACONTRATTI\\services\\backend\\user-service\\src\\__tests__\\database.integration.test.ts:43:7)","timestamp":"2025-09-24T09:20:06.485Z"}
{"code":"ENOTFOUND","errno":-3008,"hostname":"nonexistent_host","level":"error","message":"Unable to connect to database: getaddrinfo ENOTFOUND nonexistent_host","service":"user-service","stack":"Error: getaddrinfo ENOTFOUND nonexistent_host\n    at C:\\Users\\39349\\OneDrive - Poste Italiane S.p.A\\Desktop OLD\\PROGETTI VISUAL STUDIO\\PERSONALE\\FANTACONTRATTI\\node_modules\\pg-pool\\index.js:45:11\n    at processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at connectDatabase (C:\\Users\\39349\\OneDrive - Poste Italiane S.p.A\\Desktop OLD\\PROGETTI VISUAL STUDIO\\PERSONALE\\FANTACONTRATTI\\services\\backend\\user-service\\src\\database\\connection.ts:39:24)\n    at Object.<anonymous> (C:\\Users\\39349\\OneDrive - Poste Italiane S.p.A\\Desktop OLD\\PROGETTI VISUAL STUDIO\\PERSONALE\\FANTACONTRATTI\\services\\backend\\user-service\\src\\__tests__\\database.integration.test.ts:69:7)","syscall":"getaddrinfo","timestamp":"2025-09-24T09:20:12.094Z"}
{"code":"ENOTFOUND","errno":-3008,"hostname":"nonexistent_host","level":"error","message":"Unable to connect to database: getaddrinfo ENOTFOUND nonexistent_host","service":"user-service","stack":"Error: getaddrinfo ENOTFOUND nonexistent_host\n    at C:\\Users\\39349\\OneDrive - Poste Italiane S.p.A\\Desktop OLD\\PROGETTI VISUAL STUDIO\\PERSONALE\\FANTACONTRATTI\\node_modules\\pg-pool\\index.js:45:11\n    at processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at connectDatabase (C:\\Users\\39349\\OneDrive - Poste Italiane S.p.A\\Desktop OLD\\PROGETTI VISUAL STUDIO\\PERSONALE\\FANTACONTRATTI\\services\\backend\\user-service\\src\\database\\connection.ts:39:24)\n    at Object.<anonymous> (C:\\Users\\39349\\OneDrive - Poste Italiane S.p.A\\Desktop OLD\\PROGETTI VISUAL STUDIO\\PERSONALE\\FANTACONTRATTI\\services\\backend\\user-service\\src\\__tests__\\database.integration.test.ts:78:7)","syscall":"getaddrinfo","timestamp":"2025-09-24T09:20:17.566Z"}
{"code":"ENOTFOUND","errno":-3008,"hostname":"nonexistent_host","level":"error","message":"Unable to connect to database: getaddrinfo ENOTFOUND nonexistent_host","service":"user-service","stack":"Error: getaddrinfo ENOTFOUND nonexistent_host\n    at C:\\Users\\39349\\OneDrive - Poste Italiane S.p.A\\Desktop OLD\\PROGETTI VISUAL STUDIO\\PERSONALE\\FANTACONTRATTI\\node_modules\\pg-pool\\index.js:45:11\n    at processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at connectDatabase (C:\\Users\\39349\\OneDrive - Poste Italiane S.p.A\\Desktop OLD\\PROGETTI VISUAL STUDIO\\PERSONALE\\FANTACONTRATTI\\services\\backend\\user-service\\src\\database\\connection.ts:39:24)\n    at Object.<anonymous> (C:\\Users\\39349\\OneDrive - Poste Italiane S.p.A\\Desktop OLD\\PROGETTI VISUAL STUDIO\\PERSONALE\\FANTACONTRATTI\\services\\backend\\user-service\\src\\__tests__\\database.integration.test.ts:78:7)","syscall":"getaddrinfo","timestamp":"2025-09-24T09:20:23.038Z"}
{"code":"28P01","file":"auth.c","length":104,"level":"error","line":"324","message":"Unable to connect to database: password authentication failed for user \"postgres\"","name":"error","routine":"auth_failed","service":"user-service","severity":"FATAL","stack":"error: password authentication failed for user \"postgres\"\n    at C:\\Users\\39349\\OneDrive - Poste Italiane S.p.A\\Desktop OLD\\PROGETTI VISUAL STUDIO\\PERSONALE\\FANTACONTRATTI\\node_modules\\pg-pool\\index.js:45:11\n    at processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at connectDatabase (C:\\Users\\39349\\OneDrive - Poste Italiane S.p.A\\Desktop OLD\\PROGETTI VISUAL STUDIO\\PERSONALE\\FANTACONTRATTI\\services\\backend\\user-service\\src\\database\\connection.ts:52:24)\n    at Object.<anonymous> (C:\\Users\\39349\\OneDrive - Poste Italiane S.p.A\\Desktop OLD\\PROGETTI VISUAL STUDIO\\PERSONALE\\FANTACONTRATTI\\services\\backend\\user-service\\src\\__tests__\\database.integration.test.ts:18:7)","timestamp":"2025-09-24T12:28:55.739Z"}
{"code":"28P01","file":"auth.c","length":104,"level":"error","line":"324","message":"Unable to connect to database: password authentication failed for user \"postgres\"","name":"error","routine":"auth_failed","service":"user-service","severity":"FATAL","stack":"error: password authentication failed for user \"postgres\"\n    at C:\\Users\\39349\\OneDrive - Poste Italiane S.p.A\\Desktop OLD\\PROGETTI VISUAL STUDIO\\PERSONALE\\FANTACONTRATTI\\node_modules\\pg-pool\\index.js:45:11\n    at processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at connectDatabase (C:\\Users\\39349\\OneDrive - Poste Italiane S.p.A\\Desktop OLD\\PROGETTI VISUAL STUDIO\\PERSONALE\\FANTACONTRATTI\\services\\backend\\user-service\\src\\database\\connection.ts:52:24)\n    at Object.<anonymous> (C:\\Users\\39349\\OneDrive - Poste Italiane S.p.A\\Desktop OLD\\PROGETTI VISUAL STUDIO\\PERSONALE\\FANTACONTRATTI\\services\\backend\\user-service\\src\\__tests__\\database.integration.test.ts:28:7)","timestamp":"2025-09-24T12:28:58.015Z"}
{"code":"28P01","file":"auth.c","length":104,"level":"error","line":"324","message":"Unable to connect to database: password authentication failed for user \"postgres\"","name":"error","routine":"auth_failed","service":"user-service","severity":"FATAL","stack":"error: password authentication failed for user \"postgres\"\n    at C:\\Users\\39349\\OneDrive - Poste Italiane S.p.A\\Desktop OLD\\PROGETTI VISUAL STUDIO\\PERSONALE\\FANTACONTRATTI\\node_modules\\pg-pool\\index.js:45:11\n    at processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at connectDatabase (C:\\Users\\39349\\OneDrive - Poste Italiane S.p.A\\Desktop OLD\\PROGETTI VISUAL STUDIO\\PERSONALE\\FANTACONTRATTI\\services\\backend\\user-service\\src\\database\\connection.ts:52:24)\n    at Object.<anonymous> (C:\\Users\\39349\\OneDrive - Poste Italiane S.p.A\\Desktop OLD\\PROGETTI VISUAL STUDIO\\PERSONALE\\FANTACONTRATTI\\services\\backend\\user-service\\src\\__tests__\\database.integration.test.ts:43:7)","timestamp":"2025-09-24T12:29:00.167Z"}
{"code":"ENOTFOUND","errno":-3008,"hostname":"nonexistent_host","level":"error","message":"Unable to connect to database: getaddrinfo ENOTFOUND nonexistent_host","service":"user-service","stack":"Error: getaddrinfo ENOTFOUND nonexistent_host\n    at C:\\Users\\39349\\OneDrive - Poste Italiane S.p.A\\Desktop OLD\\PROGETTI VISUAL STUDIO\\PERSONALE\\FANTACONTRATTI\\node_modules\\pg-pool\\index.js:45:11\n    at processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at connectDatabase (C:\\Users\\39349\\OneDrive - Poste Italiane S.p.A\\Desktop OLD\\PROGETTI VISUAL STUDIO\\PERSONALE\\FANTACONTRATTI\\services\\backend\\user-service\\src\\database\\connection.ts:52:24)\n    at Object.<anonymous> (C:\\Users\\39349\\OneDrive - Poste Italiane S.p.A\\Desktop OLD\\PROGETTI VISUAL STUDIO\\PERSONALE\\FANTACONTRATTI\\services\\backend\\user-service\\src\\__tests__\\database.integration.test.ts:69:7)","syscall":"getaddrinfo","timestamp":"2025-09-24T12:29:05.763Z"}
{"code":"ENOTFOUND","errno":-3008,"hostname":"nonexistent_host","level":"error","message":"Unable to connect to database: getaddrinfo ENOTFOUND nonexistent_host","service":"user-service","stack":"Error: getaddrinfo ENOTFOUND nonexistent_host\n    at C:\\Users\\39349\\OneDrive - Poste Italiane S.p.A\\Desktop OLD\\PROGETTI VISUAL STUDIO\\PERSONALE\\FANTACONTRATTI\\node_modules\\pg-pool\\index.js:45:11\n    at processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at connectDatabase (C:\\Users\\39349\\OneDrive - Poste Italiane S.p.A\\Desktop OLD\\PROGETTI VISUAL STUDIO\\PERSONALE\\FANTACONTRATTI\\services\\backend\\user-service\\src\\database\\connection.ts:52:24)\n    at Object.<anonymous> (C:\\Users\\39349\\OneDrive - Poste Italiane S.p.A\\Desktop OLD\\PROGETTI VISUAL STUDIO\\PERSONALE\\FANTACONTRATTI\\services\\backend\\user-service\\src\\__tests__\\database.integration.test.ts:78:7)","syscall":"getaddrinfo","timestamp":"2025-09-24T12:29:11.202Z"}
{"code":"ENOTFOUND","errno":-3008,"hostname":"nonexistent_host","level":"error","message":"Unable to connect to database: getaddrinfo ENOTFOUND nonexistent_host","service":"user-service","stack":"Error: getaddrinfo ENOTFOUND nonexistent_host\n    at C:\\Users\\39349\\OneDrive - Poste Italiane S.p.A\\Desktop OLD\\PROGETTI VISUAL STUDIO\\PERSONALE\\FANTACONTRATTI\\node_modules\\pg-pool\\index.js:45:11\n    at processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at connectDatabase (C:\\Users\\39349\\OneDrive - Poste Italiane S.p.A\\Desktop OLD\\PROGETTI VISUAL STUDIO\\PERSONALE\\FANTACONTRATTI\\services\\backend\\user-service\\src\\database\\connection.ts:52:24)\n    at Object.<anonymous> (C:\\Users\\39349\\OneDrive - Poste Italiane S.p.A\\Desktop OLD\\PROGETTI VISUAL STUDIO\\PERSONALE\\FANTACONTRATTI\\services\\backend\\user-service\\src\\__tests__\\database.integration.test.ts:78:7)","syscall":"getaddrinfo","timestamp":"2025-09-24T12:29:16.670Z"}
{"code":"28P01","file":"auth.c","length":104,"level":"error","line":"324","message":"Unable to connect to database: password authentication failed for user \"postgres\"","name":"error","routine":"auth_failed","service":"user-service","severity":"FATAL","stack":"error: password authentication failed for user \"postgres\"\n    at C:\\Users\\39349\\OneDrive - Poste Italiane S.p.A\\Desktop OLD\\PROGETTI VISUAL STUDIO\\PERSONALE\\FANTACONTRATTI\\node_modules\\pg-pool\\index.js:45:11\n    at processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at connectDatabase (C:\\Users\\39349\\OneDrive - Poste Italiane S.p.A\\Desktop OLD\\PROGETTI VISUAL STUDIO\\PERSONALE\\FANTACONTRATTI\\services\\backend\\user-service\\src\\database\\connection.ts:52:24)\n    at Object.<anonymous> (C:\\Users\\39349\\OneDrive - Poste Italiane S.p.A\\Desktop OLD\\PROGETTI VISUAL STUDIO\\PERSONALE\\FANTACONTRATTI\\services\\backend\\user-service\\src\\__tests__\\database.integration.test.ts:18:7)","timestamp":"2025-09-24T12:34:35.847Z"}
{"code":"28P01","file":"auth.c","length":104,"level":"error","line":"324","message":"Unable to connect to database: password authentication failed for user \"postgres\"","name":"error","routine":"auth_failed","service":"user-service","severity":"FATAL","stack":"error: password authentication failed for user \"postgres\"\n    at C:\\Users\\39349\\OneDrive - Poste Italiane S.p.A\\Desktop OLD\\PROGETTI VISUAL STUDIO\\PERSONALE\\FANTACONTRATTI\\node_modules\\pg-pool\\index.js:45:11\n    at processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at connectDatabase (C:\\Users\\39349\\OneDrive - Poste Italiane S.p.A\\Desktop OLD\\PROGETTI VISUAL STUDIO\\PERSONALE\\FANTACONTRATTI\\services\\backend\\user-service\\src\\database\\connection.ts:52:24)\n    at Object.<anonymous> (C:\\Users\\39349\\OneDrive - Poste Italiane S.p.A\\Desktop OLD\\PROGETTI VISUAL STUDIO\\PERSONALE\\FANTACONTRATTI\\services\\backend\\user-service\\src\\__tests__\\database.integration.test.ts:28:7)","timestamp":"2025-09-24T12:34:38.027Z"}
{"code":"28P01","file":"auth.c","length":104,"level":"error","line":"324","message":"Unable to connect to database: password authentication failed for user \"postgres\"","name":"error","routine":"auth_failed","service":"user-service","severity":"FATAL","stack":"error: password authentication failed for user \"postgres\"\n    at C:\\Users\\39349\\OneDrive - Poste Italiane S.p.A\\Desktop OLD\\PROGETTI VISUAL STUDIO\\PERSONALE\\FANTACONTRATTI\\node_modules\\pg-pool\\index.js:45:11\n    at processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at connectDatabase (C:\\Users\\39349\\OneDrive - Poste Italiane S.p.A\\Desktop OLD\\PROGETTI VISUAL STUDIO\\PERSONALE\\FANTACONTRATTI\\services\\backend\\user-service\\src\\database\\connection.ts:52:24)\n    at Object.<anonymous> (C:\\Users\\39349\\OneDrive - Poste Italiane S.p.A\\Desktop OLD\\PROGETTI VISUAL STUDIO\\PERSONALE\\FANTACONTRATTI\\services\\backend\\user-service\\src\\__tests__\\database.integration.test.ts:43:7)","timestamp":"2025-09-24T12:34:40.307Z"}
{"code":"ENOTFOUND","errno":-3008,"hostname":"nonexistent_host","level":"error","message":"Unable to connect to database: getaddrinfo ENOTFOUND nonexistent_host","service":"user-service","stack":"Error: getaddrinfo ENOTFOUND nonexistent_host\n    at C:\\Users\\39349\\OneDrive - Poste Italiane S.p.A\\Desktop OLD\\PROGETTI VISUAL STUDIO\\PERSONALE\\FANTACONTRATTI\\node_modules\\pg-pool\\index.js:45:11\n    at processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at connectDatabase (C:\\Users\\39349\\OneDrive - Poste Italiane S.p.A\\Desktop OLD\\PROGETTI VISUAL STUDIO\\PERSONALE\\FANTACONTRATTI\\services\\backend\\user-service\\src\\database\\connection.ts:52:24)\n    at Object.<anonymous> (C:\\Users\\39349\\OneDrive - Poste Italiane S.p.A\\Desktop OLD\\PROGETTI VISUAL STUDIO\\PERSONALE\\FANTACONTRATTI\\services\\backend\\user-service\\src\\__tests__\\database.integration.test.ts:69:7)","syscall":"getaddrinfo","timestamp":"2025-09-24T12:34:45.900Z"}
{"code":"ENOTFOUND","errno":-3008,"hostname":"nonexistent_host","level":"error","message":"Unable to connect to database: getaddrinfo ENOTFOUND nonexistent_host","service":"user-service","stack":"Error: getaddrinfo ENOTFOUND nonexistent_host\n    at C:\\Users\\39349\\OneDrive - Poste Italiane S.p.A\\Desktop OLD\\PROGETTI VISUAL STUDIO\\PERSONALE\\FANTACONTRATTI\\node_modules\\pg-pool\\index.js:45:11\n    at processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at connectDatabase (C:\\Users\\39349\\OneDrive - Poste Italiane S.p.A\\Desktop OLD\\PROGETTI VISUAL STUDIO\\PERSONALE\\FANTACONTRATTI\\services\\backend\\user-service\\src\\database\\connection.ts:52:24)\n    at Object.<anonymous> (C:\\Users\\39349\\OneDrive - Poste Italiane S.p.A\\Desktop OLD\\PROGETTI VISUAL STUDIO\\PERSONALE\\FANTACONTRATTI\\services\\backend\\user-service\\src\\__tests__\\database.integration.test.ts:78:7)","syscall":"getaddrinfo","timestamp":"2025-09-24T12:34:51.370Z"}
{"code":"ENOTFOUND","errno":-3008,"hostname":"nonexistent_host","level":"error","message":"Unable to connect to database: getaddrinfo ENOTFOUND nonexistent_host","service":"user-service","stack":"Error: getaddrinfo ENOTFOUND nonexistent_host\n    at C:\\Users\\39349\\OneDrive - Poste Italiane S.p.A\\Desktop OLD\\PROGETTI VISUAL STUDIO\\PERSONALE\\FANTACONTRATTI\\node_modules\\pg-pool\\index.js:45:11\n    at processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at connectDatabase (C:\\Users\\39349\\OneDrive - Poste Italiane S.p.A\\Desktop OLD\\PROGETTI VISUAL STUDIO\\PERSONALE\\FANTACONTRATTI\\services\\backend\\user-service\\src\\database\\connection.ts:52:24)\n    at Object.<anonymous> (C:\\Users\\39349\\OneDrive - Poste Italiane S.p.A\\Desktop OLD\\PROGETTI VISUAL STUDIO\\PERSONALE\\FANTACONTRATTI\\services\\backend\\user-service\\src\\__tests__\\database.integration.test.ts:78:7)","syscall":"getaddrinfo","timestamp":"2025-09-24T12:34:56.831Z"}
{"code":"28P01","file":"auth.c","length":104,"level":"error","line":"324","message":"Unable to connect to database: password authentication failed for user \"postgres\"","name":"error","routine":"auth_failed","service":"user-service","severity":"FATAL","stack":"error: password authentication failed for user \"postgres\"\n    at C:\\Users\\39349\\OneDrive - Poste Italiane S.p.A\\Desktop OLD\\PROGETTI VISUAL STUDIO\\PERSONALE\\FANTACONTRATTI\\node_modules\\pg-pool\\index.js:45:11\n    at processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at connectDatabase (C:\\Users\\39349\\OneDrive - Poste Italiane S.p.A\\Desktop OLD\\PROGETTI VISUAL STUDIO\\PERSONALE\\FANTACONTRATTI\\services\\backend\\user-service\\src\\database\\connection.ts:52:24)\n    at Object.<anonymous> (C:\\Users\\39349\\OneDrive - Poste Italiane S.p.A\\Desktop OLD\\PROGETTI VISUAL STUDIO\\PERSONALE\\FANTACONTRATTI\\services\\backend\\user-service\\src\\__tests__\\database.integration.test.ts:18:7)","timestamp":"2025-09-24T12:45:11.293Z"}
{"code":"28P01","file":"auth.c","length":104,"level":"error","line":"324","message":"Unable to connect to database: password authentication failed for user \"postgres\"","name":"error","routine":"auth_failed","service":"user-service","severity":"FATAL","stack":"error: password authentication failed for user \"postgres\"\n    at C:\\Users\\39349\\OneDrive - Poste Italiane S.p.A\\Desktop OLD\\PROGETTI VISUAL STUDIO\\PERSONALE\\FANTACONTRATTI\\node_modules\\pg-pool\\index.js:45:11\n    at processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at connectDatabase (C:\\Users\\39349\\OneDrive - Poste Italiane S.p.A\\Desktop OLD\\PROGETTI VISUAL STUDIO\\PERSONALE\\FANTACONTRATTI\\services\\backend\\user-service\\src\\database\\connection.ts:52:24)\n    at Object.<anonymous> (C:\\Users\\39349\\OneDrive - Poste Italiane S.p.A\\Desktop OLD\\PROGETTI VISUAL STUDIO\\PERSONALE\\FANTACONTRATTI\\services\\backend\\user-service\\src\\__tests__\\database.integration.test.ts:28:7)","timestamp":"2025-09-24T12:45:13.530Z"}
{"code":"28P01","file":"auth.c","length":104,"level":"error","line":"324","message":"Unable to connect to database: password authentication failed for user \"postgres\"","name":"error","routine":"auth_failed","service":"user-service","severity":"FATAL","stack":"error: password authentication failed for user \"postgres\"\n    at C:\\Users\\39349\\OneDrive - Poste Italiane S.p.A\\Desktop OLD\\PROGETTI VISUAL STUDIO\\PERSONALE\\FANTACONTRATTI\\node_modules\\pg-pool\\index.js:45:11\n    at processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at connectDatabase (C:\\Users\\39349\\OneDrive - Poste Italiane S.p.A\\Desktop OLD\\PROGETTI VISUAL STUDIO\\PERSONALE\\FANTACONTRATTI\\services\\backend\\user-service\\src\\database\\connection.ts:52:24)\n    at Object.<anonymous> (C:\\Users\\39349\\OneDrive - Poste Italiane S.p.A\\Desktop OLD\\PROGETTI VISUAL STUDIO\\PERSONALE\\FANTACONTRATTI\\services\\backend\\user-service\\src\\__tests__\\database.integration.test.ts:43:7)","timestamp":"2025-09-24T12:45:15.784Z"}
{"code":"ENOTFOUND","errno":-3008,"hostname":"nonexistent_host","level":"error","message":"Unable to connect to database: getaddrinfo ENOTFOUND nonexistent_host","service":"user-service","stack":"Error: getaddrinfo ENOTFOUND nonexistent_host\n    at C:\\Users\\39349\\OneDrive - Poste Italiane S.p.A\\Desktop OLD\\PROGETTI VISUAL STUDIO\\PERSONALE\\FANTACONTRATTI\\node_modules\\pg-pool\\index.js:45:11\n    at processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at connectDatabase (C:\\Users\\39349\\OneDrive - Poste Italiane S.p.A\\Desktop OLD\\PROGETTI VISUAL STUDIO\\PERSONALE\\FANTACONTRATTI\\services\\backend\\user-service\\src\\database\\connection.ts:52:24)\n    at Object.<anonymous> (C:\\Users\\39349\\OneDrive - Poste Italiane S.p.A\\Desktop OLD\\PROGETTI VISUAL STUDIO\\PERSONALE\\FANTACONTRATTI\\services\\backend\\user-service\\src\\__tests__\\database.integration.test.ts:69:7)","syscall":"getaddrinfo","timestamp":"2025-09-24T12:45:21.391Z"}
{"code":"ENOTFOUND","errno":-3008,"hostname":"nonexistent_host","level":"error","message":"Unable to connect to database: getaddrinfo ENOTFOUND nonexistent_host","service":"user-service","stack":"Error: getaddrinfo ENOTFOUND nonexistent_host\n    at C:\\Users\\39349\\OneDrive - Poste Italiane S.p.A\\Desktop OLD\\PROGETTI VISUAL STUDIO\\PERSONALE\\FANTACONTRATTI\\node_modules\\pg-pool\\index.js:45:11\n    at processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at connectDatabase (C:\\Users\\39349\\OneDrive - Poste Italiane S.p.A\\Desktop OLD\\PROGETTI VISUAL STUDIO\\PERSONALE\\FANTACONTRATTI\\services\\backend\\user-service\\src\\database\\connection.ts:52:24)\n    at Object.<anonymous> (C:\\Users\\39349\\OneDrive - Poste Italiane S.p.A\\Desktop OLD\\PROGETTI VISUAL STUDIO\\PERSONALE\\FANTACONTRATTI\\services\\backend\\user-service\\src\\__tests__\\database.integration.test.ts:78:7)","syscall":"getaddrinfo","timestamp":"2025-09-24T12:45:26.864Z"}
{"code":"ENOTFOUND","errno":-3008,"hostname":"nonexistent_host","level":"error","message":"Unable to connect to database: getaddrinfo ENOTFOUND nonexistent_host","service":"user-service","stack":"Error: getaddrinfo ENOTFOUND nonexistent_host\n    at C:\\Users\\39349\\OneDrive - Poste Italiane S.p.A\\Desktop OLD\\PROGETTI VISUAL STUDIO\\PERSONALE\\FANTACONTRATTI\\node_modules\\pg-pool\\index.js:45:11\n    at processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at connectDatabase (C:\\Users\\39349\\OneDrive - Poste Italiane S.p.A\\Desktop OLD\\PROGETTI VISUAL STUDIO\\PERSONALE\\FANTACONTRATTI\\services\\backend\\user-service\\src\\database\\connection.ts:52:24)\n    at Object.<anonymous> (C:\\Users\\39349\\OneDrive - Poste Italiane S.p.A\\Desktop OLD\\PROGETTI VISUAL STUDIO\\PERSONALE\\FANTACONTRATTI\\services\\backend\\user-service\\src\\__tests__\\database.integration.test.ts:78:7)","syscall":"getaddrinfo","timestamp":"2025-09-24T12:45:32.342Z"}
{"code":"28P01","file":"auth.c","length":104,"level":"error","line":"324","message":"Unable to connect to database: password authentication failed for user \"postgres\"","name":"error","routine":"auth_failed","service":"user-service","severity":"FATAL","stack":"error: password authentication failed for user \"postgres\"\n    at C:\\Users\\39349\\OneDrive - Poste Italiane S.p.A\\Desktop OLD\\PROGETTI VISUAL STUDIO\\PERSONALE\\FANTACONTRATTI\\node_modules\\pg-pool\\index.js:45:11\n    at processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at connectDatabase (C:\\Users\\39349\\OneDrive - Poste Italiane S.p.A\\Desktop OLD\\PROGETTI VISUAL STUDIO\\PERSONALE\\FANTACONTRATTI\\services\\backend\\user-service\\src\\database\\connection.ts:54:24)\n    at Object.<anonymous> (C:\\Users\\39349\\OneDrive - Poste Italiane S.p.A\\Desktop OLD\\PROGETTI VISUAL STUDIO\\PERSONALE\\FANTACONTRATTI\\services\\backend\\user-service\\src\\__tests__\\database.integration.test.ts:18:7)","timestamp":"2025-09-24T12:57:40.374Z"}
{"code":"28P01","file":"auth.c","length":104,"level":"error","line":"324","message":"Unable to connect to database: password authentication failed for user \"postgres\"","name":"error","routine":"auth_failed","service":"user-service","severity":"FATAL","stack":"error: password authentication failed for user \"postgres\"\n    at C:\\Users\\39349\\OneDrive - Poste Italiane S.p.A\\Desktop OLD\\PROGETTI VISUAL STUDIO\\PERSONALE\\FANTACONTRATTI\\node_modules\\pg-pool\\index.js:45:11\n    at processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at connectDatabase (C:\\Users\\39349\\OneDrive - Poste Italiane S.p.A\\Desktop OLD\\PROGETTI VISUAL STUDIO\\PERSONALE\\FANTACONTRATTI\\services\\backend\\user-service\\src\\database\\connection.ts:54:24)\n    at Object.<anonymous> (C:\\Users\\39349\\OneDrive - Poste Italiane S.p.A\\Desktop OLD\\PROGETTI VISUAL STUDIO\\PERSONALE\\FANTACONTRATTI\\services\\backend\\user-service\\src\\__tests__\\database.integration.test.ts:28:7)","timestamp":"2025-09-24T12:57:42.540Z"}
{"code":"28P01","file":"auth.c","length":104,"level":"error","line":"324","message":"Unable to connect to database: password authentication failed for user \"postgres\"","name":"error","routine":"auth_failed","service":"user-service","severity":"FATAL","stack":"error: password authentication failed for user \"postgres\"\n    at C:\\Users\\39349\\OneDrive - Poste Italiane S.p.A\\Desktop OLD\\PROGETTI VISUAL STUDIO\\PERSONALE\\FANTACONTRATTI\\node_modules\\pg-pool\\index.js:45:11\n    at processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at connectDatabase (C:\\Users\\39349\\OneDrive - Poste Italiane S.p.A\\Desktop OLD\\PROGETTI VISUAL STUDIO\\PERSONALE\\FANTACONTRATTI\\services\\backend\\user-service\\src\\database\\connection.ts:54:24)\n    at Object.<anonymous> (C:\\Users\\39349\\OneDrive - Poste Italiane S.p.A\\Desktop OLD\\PROGETTI VISUAL STUDIO\\PERSONALE\\FANTACONTRATTI\\services\\backend\\user-service\\src\\__tests__\\database.integration.test.ts:43:7)","timestamp":"2025-09-24T12:57:44.713Z"}
{"code":"ENOTFOUND","errno":-3008,"hostname":"nonexistent_host","level":"error","message":"Unable to connect to database: getaddrinfo ENOTFOUND nonexistent_host","service":"user-service","stack":"Error: getaddrinfo ENOTFOUND nonexistent_host\n    at C:\\Users\\39349\\OneDrive - Poste Italiane S.p.A\\Desktop OLD\\PROGETTI VISUAL STUDIO\\PERSONALE\\FANTACONTRATTI\\node_modules\\pg-pool\\index.js:45:11\n    at processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at connectDatabase (C:\\Users\\39349\\OneDrive - Poste Italiane S.p.A\\Desktop OLD\\PROGETTI VISUAL STUDIO\\PERSONALE\\FANTACONTRATTI\\services\\backend\\user-service\\src\\database\\connection.ts:54:24)\n    at Object.<anonymous> (C:\\Users\\39349\\OneDrive - Poste Italiane S.p.A\\Desktop OLD\\PROGETTI VISUAL STUDIO\\PERSONALE\\FANTACONTRATTI\\services\\backend\\user-service\\src\\__tests__\\database.integration.test.ts:69:7)","syscall":"getaddrinfo","timestamp":"2025-09-24T12:57:50.317Z"}
{"code":"ENOTFOUND","errno":-3008,"hostname":"nonexistent_host","level":"error","message":"Unable to connect to database: getaddrinfo ENOTFOUND nonexistent_host","service":"user-service","stack":"Error: getaddrinfo ENOTFOUND nonexistent_host\n    at C:\\Users\\39349\\OneDrive - Poste Italiane S.p.A\\Desktop OLD\\PROGETTI VISUAL STUDIO\\PERSONALE\\FANTACONTRATTI\\node_modules\\pg-pool\\index.js:45:11\n    at processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at connectDatabase (C:\\Users\\39349\\OneDrive - Poste Italiane S.p.A\\Desktop OLD\\PROGETTI VISUAL STUDIO\\PERSONALE\\FANTACONTRATTI\\services\\backend\\user-service\\src\\database\\connection.ts:54:24)\n    at Object.<anonymous> (C:\\Users\\39349\\OneDrive - Poste Italiane S.p.A\\Desktop OLD\\PROGETTI VISUAL STUDIO\\PERSONALE\\FANTACONTRATTI\\services\\backend\\user-service\\src\\__tests__\\database.integration.test.ts:78:7)","syscall":"getaddrinfo","timestamp":"2025-09-24T12:57:55.772Z"}
{"code":"ENOTFOUND","errno":-3008,"hostname":"nonexistent_host","level":"error","message":"Unable to connect to database: getaddrinfo ENOTFOUND nonexistent_host","service":"user-service","stack":"Error: getaddrinfo ENOTFOUND nonexistent_host\n    at C:\\Users\\39349\\OneDrive - Poste Italiane S.p.A\\Desktop OLD\\PROGETTI VISUAL STUDIO\\PERSONALE\\FANTACONTRATTI\\node_modules\\pg-pool\\index.js:45:11\n    at processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at connectDatabase (C:\\Users\\39349\\OneDrive - Poste Italiane S.p.A\\Desktop OLD\\PROGETTI VISUAL STUDIO\\PERSONALE\\FANTACONTRATTI\\services\\backend\\user-service\\src\\database\\connection.ts:54:24)\n    at Object.<anonymous> (C:\\Users\\39349\\OneDrive - Poste Italiane S.p.A\\Desktop OLD\\PROGETTI VISUAL STUDIO\\PERSONALE\\FANTACONTRATTI\\services\\backend\\user-service\\src\\__tests__\\database.integration.test.ts:78:7)","syscall":"getaddrinfo","timestamp":"2025-09-24T12:58:01.253Z"}
{"code":"28P01","file":"auth.c","length":104,"level":"error","line":"324","message":"Unable to connect to database: password authentication failed for user \"postgres\"","name":"error","routine":"auth_failed","service":"user-service","severity":"FATAL","stack":"error: password authentication failed for user \"postgres\"\n    at C:\\Users\\39349\\OneDrive - Poste Italiane S.p.A\\Desktop OLD\\PROGETTI VISUAL STUDIO\\PERSONALE\\FANTACONTRATTI\\node_modules\\pg-pool\\index.js:45:11\n    at processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at connectDatabase (C:\\Users\\39349\\OneDrive - Poste Italiane S.p.A\\Desktop OLD\\PROGETTI VISUAL STUDIO\\PERSONALE\\FANTACONTRATTI\\services\\backend\\user-service\\src\\database\\connection.ts:54:24)\n    at Object.<anonymous> (C:\\Users\\39349\\OneDrive - Poste Italiane S.p.A\\Desktop OLD\\PROGETTI VISUAL STUDIO\\PERSONALE\\FANTACONTRATTI\\services\\backend\\user-service\\src\\__tests__\\database.integration.test.ts:18:7)","timestamp":"2025-09-24T12:59:38.650Z"}
{"code":"28P01","file":"auth.c","length":104,"level":"error","line":"324","message":"Unable to connect to database: password authentication failed for user \"postgres\"","name":"error","routine":"auth_failed","service":"user-service","severity":"FATAL","stack":"error: password authentication failed for user \"postgres\"\n    at C:\\Users\\39349\\OneDrive - Poste Italiane S.p.A\\Desktop OLD\\PROGETTI VISUAL STUDIO\\PERSONALE\\FANTACONTRATTI\\node_modules\\pg-pool\\index.js:45:11\n    at processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at connectDatabase (C:\\Users\\39349\\OneDrive - Poste Italiane S.p.A\\Desktop OLD\\PROGETTI VISUAL STUDIO\\PERSONALE\\FANTACONTRATTI\\services\\backend\\user-service\\src\\database\\connection.ts:54:24)\n    at Object.<anonymous> (C:\\Users\\39349\\OneDrive - Poste Italiane S.p.A\\Desktop OLD\\PROGETTI VISUAL STUDIO\\PERSONALE\\FANTACONTRATTI\\services\\backend\\user-service\\src\\__tests__\\database.integration.test.ts:28:7)","timestamp":"2025-09-24T12:59:40.819Z"}
{"code":"28P01","file":"auth.c","length":104,"level":"error","line":"324","message":"Unable to connect to database: password authentication failed for user \"postgres\"","name":"error","routine":"auth_failed","service":"user-service","severity":"FATAL","stack":"error: password authentication failed for user \"postgres\"\n    at C:\\Users\\39349\\OneDrive - Poste Italiane S.p.A\\Desktop OLD\\PROGETTI VISUAL STUDIO\\PERSONALE\\FANTACONTRATTI\\node_modules\\pg-pool\\index.js:45:11\n    at processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at connectDatabase (C:\\Users\\39349\\OneDrive - Poste Italiane S.p.A\\Desktop OLD\\PROGETTI VISUAL STUDIO\\PERSONALE\\FANTACONTRATTI\\services\\backend\\user-service\\src\\database\\connection.ts:54:24)\n    at Object.<anonymous> (C:\\Users\\39349\\OneDrive - Poste Italiane S.p.A\\Desktop OLD\\PROGETTI VISUAL STUDIO\\PERSONALE\\FANTACONTRATTI\\services\\backend\\user-service\\src\\__tests__\\database.integration.test.ts:43:7)","timestamp":"2025-09-24T12:59:42.984Z"}
{"code":"ENOTFOUND","errno":-3008,"hostname":"nonexistent_host","level":"error","message":"Unable to connect to database: getaddrinfo ENOTFOUND nonexistent_host","service":"user-service","stack":"Error: getaddrinfo ENOTFOUND nonexistent_host\n    at C:\\Users\\39349\\OneDrive - Poste Italiane S.p.A\\Desktop OLD\\PROGETTI VISUAL STUDIO\\PERSONALE\\FANTACONTRATTI\\node_modules\\pg-pool\\index.js:45:11\n    at processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at connectDatabase (C:\\Users\\39349\\OneDrive - Poste Italiane S.p.A\\Desktop OLD\\PROGETTI VISUAL STUDIO\\PERSONALE\\FANTACONTRATTI\\services\\backend\\user-service\\src\\database\\connection.ts:54:24)\n    at Object.<anonymous> (C:\\Users\\39349\\OneDrive - Poste Italiane S.p.A\\Desktop OLD\\PROGETTI VISUAL STUDIO\\PERSONALE\\FANTACONTRATTI\\services\\backend\\user-service\\src\\__tests__\\database.integration.test.ts:69:7)","syscall":"getaddrinfo","timestamp":"2025-09-24T12:59:48.564Z"}
{"code":"ENOTFOUND","errno":-3008,"hostname":"nonexistent_host","level":"error","message":"Unable to connect to database: getaddrinfo ENOTFOUND nonexistent_host","service":"user-service","stack":"Error: getaddrinfo ENOTFOUND nonexistent_host\n    at C:\\Users\\39349\\OneDrive - Poste Italiane S.p.A\\Desktop OLD\\PROGETTI VISUAL STUDIO\\PERSONALE\\FANTACONTRATTI\\node_modules\\pg-pool\\index.js:45:11\n    at processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at connectDatabase (C:\\Users\\39349\\OneDrive - Poste Italiane S.p.A\\Desktop OLD\\PROGETTI VISUAL STUDIO\\PERSONALE\\FANTACONTRATTI\\services\\backend\\user-service\\src\\database\\connection.ts:54:24)\n    at Object.<anonymous> (C:\\Users\\39349\\OneDrive - Poste Italiane S.p.A\\Desktop OLD\\PROGETTI VISUAL STUDIO\\PERSONALE\\FANTACONTRATTI\\services\\backend\\user-service\\src\\__tests__\\database.integration.test.ts:78:7)","syscall":"getaddrinfo","timestamp":"2025-09-24T12:59:54.012Z"}
{"code":"ENOTFOUND","errno":-3008,"hostname":"nonexistent_host","level":"error","message":"Unable to connect to database: getaddrinfo ENOTFOUND nonexistent_host","service":"user-service","stack":"Error: getaddrinfo ENOTFOUND nonexistent_host\n    at C:\\Users\\39349\\OneDrive - Poste Italiane S.p.A\\Desktop OLD\\PROGETTI VISUAL STUDIO\\PERSONALE\\FANTACONTRATTI\\node_modules\\pg-pool\\index.js:45:11\n    at processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at connectDatabase (C:\\Users\\39349\\OneDrive - Poste Italiane S.p.A\\Desktop OLD\\PROGETTI VISUAL STUDIO\\PERSONALE\\FANTACONTRATTI\\services\\backend\\user-service\\src\\database\\connection.ts:54:24)\n    at Object.<anonymous> (C:\\Users\\39349\\OneDrive - Poste Italiane S.p.A\\Desktop OLD\\PROGETTI VISUAL STUDIO\\PERSONALE\\FANTACONTRATTI\\services\\backend\\user-service\\src\\__tests__\\database.integration.test.ts:78:7)","syscall":"getaddrinfo","timestamp":"2025-09-24T12:59:59.466Z"}
{"code":"28P01","file":"auth.c","length":104,"level":"error","line":"324","message":"Unable to connect to database: password authentication failed for user \"postgres\"","name":"error","routine":"auth_failed","service":"user-service","severity":"FATAL","stack":"error: password authentication failed for user \"postgres\"\n    at C:\\Users\\39349\\OneDrive - Poste Italiane S.p.A\\Desktop OLD\\PROGETTI VISUAL STUDIO\\PERSONALE\\FANTACONTRATTI\\node_modules\\pg-pool\\index.js:45:11\n    at processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at connectDatabase (C:\\Users\\39349\\OneDrive - Poste Italiane S.p.A\\Desktop OLD\\PROGETTI VISUAL STUDIO\\PERSONALE\\FANTACONTRATTI\\services\\backend\\user-service\\src\\database\\connection.ts:62:24)\n    at Object.<anonymous> (C:\\Users\\39349\\OneDrive - Poste Italiane S.p.A\\Desktop OLD\\PROGETTI VISUAL STUDIO\\PERSONALE\\FANTACONTRATTI\\services\\backend\\user-service\\src\\__tests__\\database.integration.test.ts:18:7)","timestamp":"2025-09-24T13:04:20.472Z"}
{"code":"28P01","file":"auth.c","length":104,"level":"error","line":"324","message":"Unable to connect to database: password authentication failed for user \"postgres\"","name":"error","routine":"auth_failed","service":"user-service","severity":"FATAL","stack":"error: password authentication failed for user \"postgres\"\n    at C:\\Users\\39349\\OneDrive - Poste Italiane S.p.A\\Desktop OLD\\PROGETTI VISUAL STUDIO\\PERSONALE\\FANTACONTRATTI\\node_modules\\pg-pool\\index.js:45:11\n    at processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at connectDatabase (C:\\Users\\39349\\OneDrive - Poste Italiane S.p.A\\Desktop OLD\\PROGETTI VISUAL STUDIO\\PERSONALE\\FANTACONTRATTI\\services\\backend\\user-service\\src\\database\\connection.ts:62:24)\n    at Object.<anonymous> (C:\\Users\\39349\\OneDrive - Poste Italiane S.p.A\\Desktop OLD\\PROGETTI VISUAL STUDIO\\PERSONALE\\FANTACONTRATTI\\services\\backend\\user-service\\src\\__tests__\\database.integration.test.ts:28:7)","timestamp":"2025-09-24T13:04:22.631Z"}
{"code":"28P01","file":"auth.c","length":104,"level":"error","line":"324","message":"Unable to connect to database: password authentication failed for user \"postgres\"","name":"error","routine":"auth_failed","service":"user-service","severity":"FATAL","stack":"error: password authentication failed for user \"postgres\"\n    at C:\\Users\\39349\\OneDrive - Poste Italiane S.p.A\\Desktop OLD\\PROGETTI VISUAL STUDIO\\PERSONALE\\FANTACONTRATTI\\node_modules\\pg-pool\\index.js:45:11\n    at processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at connectDatabase (C:\\Users\\39349\\OneDrive - Poste Italiane S.p.A\\Desktop OLD\\PROGETTI VISUAL STUDIO\\PERSONALE\\FANTACONTRATTI\\services\\backend\\user-service\\src\\database\\connection.ts:62:24)\n    at Object.<anonymous> (C:\\Users\\39349\\OneDrive - Poste Italiane S.p.A\\Desktop OLD\\PROGETTI VISUAL STUDIO\\PERSONALE\\FANTACONTRATTI\\services\\backend\\user-service\\src\\__tests__\\database.integration.test.ts:43:7)","timestamp":"2025-09-24T13:04:24.997Z"}
{"code":"28P01","file":"auth.c","length":104,"level":"error","line":"324","message":"Unable to connect to database: password authentication failed for user \"postgres\"","name":"error","routine":"auth_failed","service":"user-service","severity":"FATAL","stack":"error: password authentication failed for user \"postgres\"\n    at C:\\Users\\39349\\OneDrive - Poste Italiane S.p.A\\Desktop OLD\\PROGETTI VISUAL STUDIO\\PERSONALE\\FANTACONTRATTI\\node_modules\\pg-pool\\index.js:45:11\n    at processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at connectDatabase (C:\\Users\\39349\\OneDrive - Poste Italiane S.p.A\\Desktop OLD\\PROGETTI VISUAL STUDIO\\PERSONALE\\FANTACONTRATTI\\services\\backend\\user-service\\src\\database\\connection.ts:62:24)\n    at Object.<anonymous> (C:\\Users\\39349\\OneDrive - Poste Italiane S.p.A\\Desktop OLD\\PROGETTI VISUAL STUDIO\\PERSONALE\\FANTACONTRATTI\\services\\backend\\user-service\\src\\__tests__\\database.integration.test.ts:69:7)","timestamp":"2025-09-24T13:04:27.167Z"}
{"code":"28P01","file":"auth.c","length":104,"level":"error","line":"324","message":"Unable to connect to database: password authentication failed for user \"postgres\"","name":"error","routine":"auth_failed","service":"user-service","severity":"FATAL","stack":"error: password authentication failed for user \"postgres\"\n    at C:\\Users\\39349\\OneDrive - Poste Italiane S.p.A\\Desktop OLD\\PROGETTI VISUAL STUDIO\\PERSONALE\\FANTACONTRATTI\\node_modules\\pg-pool\\index.js:45:11\n    at processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at connectDatabase (C:\\Users\\39349\\OneDrive - Poste Italiane S.p.A\\Desktop OLD\\PROGETTI VISUAL STUDIO\\PERSONALE\\FANTACONTRATTI\\services\\backend\\user-service\\src\\database\\connection.ts:62:24)\n    at Object.<anonymous> (C:\\Users\\39349\\OneDrive - Poste Italiane S.p.A\\Desktop OLD\\PROGETTI VISUAL STUDIO\\PERSONALE\\FANTACONTRATTI\\services\\backend\\user-service\\src\\__tests__\\database.integration.test.ts:78:7)","timestamp":"2025-09-24T13:04:29.322Z"}
{"code":"28P01","file":"auth.c","length":104,"level":"error","line":"324","message":"Unable to connect to database: password authentication failed for user \"postgres\"","name":"error","routine":"auth_failed","service":"user-service","severity":"FATAL","stack":"error: password authentication failed for user \"postgres\"\n    at C:\\Users\\39349\\OneDrive - Poste Italiane S.p.A\\Desktop OLD\\PROGETTI VISUAL STUDIO\\PERSONALE\\FANTACONTRATTI\\node_modules\\pg-pool\\index.js:45:11\n    at processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at connectDatabase (C:\\Users\\39349\\OneDrive - Poste Italiane S.p.A\\Desktop OLD\\PROGETTI VISUAL STUDIO\\PERSONALE\\FANTACONTRATTI\\services\\backend\\user-service\\src\\database\\connection.ts:62:24)\n    at Object.<anonymous> (C:\\Users\\39349\\OneDrive - Poste Italiane S.p.A\\Desktop OLD\\PROGETTI VISUAL STUDIO\\PERSONALE\\FANTACONTRATTI\\services\\backend\\user-service\\src\\__tests__\\database.integration.test.ts:78:7)","timestamp":"2025-09-24T13:04:31.482Z"}
{"code":"28P01","file":"auth.c","length":104,"level":"error","line":"324","message":"Unable to connect to database: password authentication failed for user \"postgres\"","name":"error","routine":"auth_failed","service":"user-service","severity":"FATAL","stack":"error: password authentication failed for user \"postgres\"\n    at C:\\Users\\39349\\OneDrive - Poste Italiane S.p.A\\Desktop OLD\\PROGETTI VISUAL STUDIO\\PERSONALE\\FANTACONTRATTI\\node_modules\\pg-pool\\index.js:45:11\n    at processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at connectDatabase (C:\\Users\\39349\\OneDrive - Poste Italiane S.p.A\\Desktop OLD\\PROGETTI VISUAL STUDIO\\PERSONALE\\FANTACONTRATTI\\services\\backend\\user-service\\src\\database\\connection.ts:65:24)\n    at Object.<anonymous> (C:\\Users\\39349\\OneDrive - Poste Italiane S.p.A\\Desktop OLD\\PROGETTI VISUAL STUDIO\\PERSONALE\\FANTACONTRATTI\\services\\backend\\user-service\\src\\__tests__\\database.integration.test.ts:18:7)","timestamp":"2025-09-24T14:19:07.374Z"}
{"code":"28P01","file":"auth.c","length":104,"level":"error","line":"324","message":"Unable to connect to database: password authentication failed for user \"postgres\"","name":"error","routine":"auth_failed","service":"user-service","severity":"FATAL","stack":"error: password authentication failed for user \"postgres\"\n    at C:\\Users\\39349\\OneDrive - Poste Italiane S.p.A\\Desktop OLD\\PROGETTI VISUAL STUDIO\\PERSONALE\\FANTACONTRATTI\\node_modules\\pg-pool\\index.js:45:11\n    at processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at connectDatabase (C:\\Users\\39349\\OneDrive - Poste Italiane S.p.A\\Desktop OLD\\PROGETTI VISUAL STUDIO\\PERSONALE\\FANTACONTRATTI\\services\\backend\\user-service\\src\\database\\connection.ts:65:24)\n    at Object.<anonymous> (C:\\Users\\39349\\OneDrive - Poste Italiane S.p.A\\Desktop OLD\\PROGETTI VISUAL STUDIO\\PERSONALE\\FANTACONTRATTI\\services\\backend\\user-service\\src\\__tests__\\database.integration.test.ts:28:7)","timestamp":"2025-09-24T14:19:09.575Z"}
{"code":"28P01","file":"auth.c","length":104,"level":"error","line":"324","message":"Unable to connect to database: password authentication failed for user \"postgres\"","name":"error","routine":"auth_failed","service":"user-service","severity":"FATAL","stack":"error: password authentication failed for user \"postgres\"\n    at C:\\Users\\39349\\OneDrive - Poste Italiane S.p.A\\Desktop OLD\\PROGETTI VISUAL STUDIO\\PERSONALE\\FANTACONTRATTI\\node_modules\\pg-pool\\index.js:45:11\n    at processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at connectDatabase (C:\\Users\\39349\\OneDrive - Poste Italiane S.p.A\\Desktop OLD\\PROGETTI VISUAL STUDIO\\PERSONALE\\FANTACONTRATTI\\services\\backend\\user-service\\src\\database\\connection.ts:65:24)\n    at Object.<anonymous> (C:\\Users\\39349\\OneDrive - Poste Italiane S.p.A\\Desktop OLD\\PROGETTI VISUAL STUDIO\\PERSONALE\\FANTACONTRATTI\\services\\backend\\user-service\\src\\__tests__\\database.integration.test.ts:43:7)","timestamp":"2025-09-24T14:19:11.847Z"}
{"code":"28P01","file":"auth.c","length":104,"level":"error","line":"324","message":"Unable to connect to database: password authentication failed for user \"postgres\"","name":"error","routine":"auth_failed","service":"user-service","severity":"FATAL","stack":"error: password authentication failed for user \"postgres\"\n    at C:\\Users\\39349\\OneDrive - Poste Italiane S.p.A\\Desktop OLD\\PROGETTI VISUAL STUDIO\\PERSONALE\\FANTACONTRATTI\\node_modules\\pg-pool\\index.js:45:11\n    at processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at connectDatabase (C:\\Users\\39349\\OneDrive - Poste Italiane S.p.A\\Desktop OLD\\PROGETTI VISUAL STUDIO\\PERSONALE\\FANTACONTRATTI\\services\\backend\\user-service\\src\\database\\connection.ts:65:24)\n    at Object.<anonymous> (C:\\Users\\39349\\OneDrive - Poste Italiane S.p.A\\Desktop OLD\\PROGETTI VISUAL STUDIO\\PERSONALE\\FANTACONTRATTI\\services\\backend\\user-service\\src\\__tests__\\database.integration.test.ts:69:7)","timestamp":"2025-09-24T14:19:14.041Z"}
{"code":"28P01","file":"auth.c","length":104,"level":"error","line":"324","message":"Unable to connect to database: password authentication failed for user \"postgres\"","name":"error","routine":"auth_failed","service":"user-service","severity":"FATAL","stack":"error: password authentication failed for user \"postgres\"\n    at C:\\Users\\39349\\OneDrive - Poste Italiane S.p.A\\Desktop OLD\\PROGETTI VISUAL STUDIO\\PERSONALE\\FANTACONTRATTI\\node_modules\\pg-pool\\index.js:45:11\n    at processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at connectDatabase (C:\\Users\\39349\\OneDrive - Poste Italiane S.p.A\\Desktop OLD\\PROGETTI VISUAL STUDIO\\PERSONALE\\FANTACONTRATTI\\services\\backend\\user-service\\src\\database\\connection.ts:65:24)\n    at Object.<anonymous> (C:\\Users\\39349\\OneDrive - Poste Italiane S.p.A\\Desktop OLD\\PROGETTI VISUAL STUDIO\\PERSONALE\\FANTACONTRATTI\\services\\backend\\user-service\\src\\__tests__\\database.integration.test.ts:78:7)","timestamp":"2025-09-24T14:19:16.410Z"}
{"code":"28P01","file":"auth.c","length":104,"level":"error","line":"324","message":"Unable to connect to database: password authentication failed for user \"postgres\"","name":"error","routine":"auth_failed","service":"user-service","severity":"FATAL","stack":"error: password authentication failed for user \"postgres\"\n    at C:\\Users\\39349\\OneDrive - Poste Italiane S.p.A\\Desktop OLD\\PROGETTI VISUAL STUDIO\\PERSONALE\\FANTACONTRATTI\\node_modules\\pg-pool\\index.js:45:11\n    at processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at connectDatabase (C:\\Users\\39349\\OneDrive - Poste Italiane S.p.A\\Desktop OLD\\PROGETTI VISUAL STUDIO\\PERSONALE\\FANTACONTRATTI\\services\\backend\\user-service\\src\\database\\connection.ts:65:24)\n    at Object.<anonymous> (C:\\Users\\39349\\OneDrive - Poste Italiane S.p.A\\Desktop OLD\\PROGETTI VISUAL STUDIO\\PERSONALE\\FANTACONTRATTI\\services\\backend\\user-service\\src\\__tests__\\database.integration.test.ts:78:7)","timestamp":"2025-09-24T14:19:18.639Z"}
{"code":"28P01","file":"auth.c","length":104,"level":"error","line":"324","message":"Unable to connect to database: password authentication failed for user \"postgres\"","name":"error","routine":"auth_failed","service":"user-service","severity":"FATAL","stack":"error: password authentication failed for user \"postgres\"\n    at C:\\Users\\39349\\OneDrive - Poste Italiane S.p.A\\Desktop OLD\\PROGETTI VISUAL STUDIO\\PERSONALE\\FANTACONTRATTI\\node_modules\\pg-pool\\index.js:45:11\n    at processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at connectDatabase (C:\\Users\\39349\\OneDrive - Poste Italiane S.p.A\\Desktop OLD\\PROGETTI VISUAL STUDIO\\PERSONALE\\FANTACONTRATTI\\services\\backend\\user-service\\src\\database\\connection.ts:72:24)\n    at Object.<anonymous> (C:\\Users\\39349\\OneDrive - Poste Italiane S.p.A\\Desktop OLD\\PROGETTI VISUAL STUDIO\\PERSONALE\\FANTACONTRATTI\\services\\backend\\user-service\\src\\__tests__\\integration.database.tdd.test.ts:36:7)","timestamp":"2025-09-25T06:40:44.552Z"}
{"code":"28P01","file":"auth.c","length":104,"level":"error","line":"324","message":"Unable to connect to database: password authentication failed for user \"postgres\"","name":"error","routine":"auth_failed","service":"user-service","severity":"FATAL","stack":"error: password authentication failed for user \"postgres\"\n    at C:\\Users\\39349\\OneDrive - Poste Italiane S.p.A\\Desktop OLD\\PROGETTI VISUAL STUDIO\\PERSONALE\\FANTACONTRATTI\\node_modules\\pg-pool\\index.js:45:11\n    at processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at connectDatabase (C:\\Users\\39349\\OneDrive - Poste Italiane S.p.A\\Desktop OLD\\PROGETTI VISUAL STUDIO\\PERSONALE\\FANTACONTRATTI\\services\\backend\\user-service\\src\\database\\connection.ts:72:24)\n    at Object.<anonymous> (C:\\Users\\39349\\OneDrive - Poste Italiane S.p.A\\Desktop OLD\\PROGETTI VISUAL STUDIO\\PERSONALE\\FANTACONTRATTI\\services\\backend\\user-service\\src\\__tests__\\integration.database.tdd.test.ts:45:7)","timestamp":"2025-09-25T06:40:46.711Z"}
{"code":"28P01","file":"auth.c","length":104,"level":"error","line":"324","message":"Unable to connect to database: password authentication failed for user \"postgres\"","name":"error","routine":"auth_failed","service":"user-service","severity":"FATAL","stack":"error: password authentication failed for user \"postgres\"\n    at C:\\Users\\39349\\OneDrive - Poste Italiane S.p.A\\Desktop OLD\\PROGETTI VISUAL STUDIO\\PERSONALE\\FANTACONTRATTI\\node_modules\\pg-pool\\index.js:45:11\n    at processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at connectDatabase (C:\\Users\\39349\\OneDrive - Poste Italiane S.p.A\\Desktop OLD\\PROGETTI VISUAL STUDIO\\PERSONALE\\FANTACONTRATTI\\services\\backend\\user-service\\src\\database\\connection.ts:72:24)\n    at Object.<anonymous> (C:\\Users\\39349\\OneDrive - Poste Italiane S.p.A\\Desktop OLD\\PROGETTI VISUAL STUDIO\\PERSONALE\\FANTACONTRATTI\\services\\backend\\user-service\\src\\__tests__\\integration.database.tdd.test.ts:60:7)","timestamp":"2025-09-25T06:40:48.874Z"}
{"code":"28P01","file":"auth.c","length":104,"level":"error","line":"324","message":"Unable to connect to database: password authentication failed for user \"postgres\"","name":"error","routine":"auth_failed","service":"user-service","severity":"FATAL","stack":"error: password authentication failed for user \"postgres\"\n    at C:\\Users\\39349\\OneDrive - Poste Italiane S.p.A\\Desktop OLD\\PROGETTI VISUAL STUDIO\\PERSONALE\\FANTACONTRATTI\\node_modules\\pg-pool\\index.js:45:11\n    at processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at connectDatabase (C:\\Users\\39349\\OneDrive - Poste Italiane S.p.A\\Desktop OLD\\PROGETTI VISUAL STUDIO\\PERSONALE\\FANTACONTRATTI\\services\\backend\\user-service\\src\\database\\connection.ts:72:24)\n    at Object.<anonymous> (C:\\Users\\39349\\OneDrive - Poste Italiane S.p.A\\Desktop OLD\\PROGETTI VISUAL STUDIO\\PERSONALE\\FANTACONTRATTI\\services\\backend\\user-service\\src\\__tests__\\integration.database.tdd.test.ts:84:7)","timestamp":"2025-09-25T06:40:51.039Z"}
{"code":"ENOTFOUND","errno":-3008,"hostname":"nonexistent-host","level":"error","message":"Unable to connect to database: getaddrinfo ENOTFOUND nonexistent-host","service":"user-service","stack":"Error: getaddrinfo ENOTFOUND nonexistent-host\n    at C:\\Users\\39349\\OneDrive - Poste Italiane S.p.A\\Desktop OLD\\PROGETTI VISUAL STUDIO\\PERSONALE\\FANTACONTRATTI\\node_modules\\pg-pool\\index.js:45:11\n    at processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at connectDatabase (C:\\Users\\39349\\OneDrive - Poste Italiane S.p.A\\Desktop OLD\\PROGETTI VISUAL STUDIO\\PERSONALE\\FANTACONTRATTI\\services\\backend\\user-service\\src\\database\\connection.ts:72:24)\n    at Object.<anonymous> (C:\\Users\\39349\\OneDrive - Poste Italiane S.p.A\\Desktop OLD\\PROGETTI VISUAL STUDIO\\PERSONALE\\FANTACONTRATTI\\services\\backend\\user-service\\src\\__tests__\\integration.database.tdd.test.ts:157:9)","syscall":"getaddrinfo","timestamp":"2025-09-25T06:40:56.639Z"}
{"code":"28P01","file":"auth.c","length":104,"level":"error","line":"324","message":"Unable to connect to database: password authentication failed for user \"postgres\"","name":"error","routine":"auth_failed","service":"user-service","severity":"FATAL","stack":"error: password authentication failed for user \"postgres\"\n    at C:\\Users\\39349\\OneDrive - Poste Italiane S.p.A\\Desktop OLD\\PROGETTI VISUAL STUDIO\\PERSONALE\\FANTACONTRATTI\\node_modules\\pg-pool\\index.js:45:11\n    at processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at connectDatabase (C:\\Users\\39349\\OneDrive - Poste Italiane S.p.A\\Desktop OLD\\PROGETTI VISUAL STUDIO\\PERSONALE\\FANTACONTRATTI\\services\\backend\\user-service\\src\\database\\connection.ts:72:24)\n    at Object.<anonymous> (C:\\Users\\39349\\OneDrive - Poste Italiane S.p.A\\Desktop OLD\\PROGETTI VISUAL STUDIO\\PERSONALE\\FANTACONTRATTI\\services\\backend\\user-service\\src\\__tests__\\integration.database.tdd.test.ts:167:7)","timestamp":"2025-09-25T06:40:58.816Z"}
{"code":"28P01","file":"auth.c","length":104,"level":"error","line":"324","message":"Unable to connect to database: password authentication failed for user \"postgres\"","name":"error","routine":"auth_failed","service":"user-service","severity":"FATAL","stack":"error: password authentication failed for user \"postgres\"\n    at C:\\Users\\39349\\OneDrive - Poste Italiane S.p.A\\Desktop OLD\\PROGETTI VISUAL STUDIO\\PERSONALE\\FANTACONTRATTI\\node_modules\\pg-pool\\index.js:45:11\n    at processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at connectDatabase (C:\\Users\\39349\\OneDrive - Poste Italiane S.p.A\\Desktop OLD\\PROGETTI VISUAL STUDIO\\PERSONALE\\FANTACONTRATTI\\services\\backend\\user-service\\src\\database\\connection.ts:72:24)\n    at Object.<anonymous> (C:\\Users\\39349\\OneDrive - Poste Italiane S.p.A\\Desktop OLD\\PROGETTI VISUAL STUDIO\\PERSONALE\\FANTACONTRATTI\\services\\backend\\user-service\\src\\__tests__\\integration.database.tdd.test.ts:36:7)","timestamp":"2025-09-25T06:50:02.798Z"}
{"code":"28P01","file":"auth.c","length":104,"level":"error","line":"324","message":"Unable to connect to database: password authentication failed for user \"postgres\"","name":"error","routine":"auth_failed","service":"user-service","severity":"FATAL","stack":"error: password authentication failed for user \"postgres\"\n    at C:\\Users\\39349\\OneDrive - Poste Italiane S.p.A\\Desktop OLD\\PROGETTI VISUAL STUDIO\\PERSONALE\\FANTACONTRATTI\\node_modules\\pg-pool\\index.js:45:11\n    at processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at connectDatabase (C:\\Users\\39349\\OneDrive - Poste Italiane S.p.A\\Desktop OLD\\PROGETTI VISUAL STUDIO\\PERSONALE\\FANTACONTRATTI\\services\\backend\\user-service\\src\\database\\connection.ts:72:24)\n    at Object.<anonymous> (C:\\Users\\39349\\OneDrive - Poste Italiane S.p.A\\Desktop OLD\\PROGETTI VISUAL STUDIO\\PERSONALE\\FANTACONTRATTI\\services\\backend\\user-service\\src\\__tests__\\integration.database.tdd.test.ts:45:7)","timestamp":"2025-09-25T06:50:04.989Z"}
{"code":"28P01","file":"auth.c","length":104,"level":"error","line":"324","message":"Unable to connect to database: password authentication failed for user \"postgres\"","name":"error","routine":"auth_failed","service":"user-service","severity":"FATAL","stack":"error: password authentication failed for user \"postgres\"\n    at C:\\Users\\39349\\OneDrive - Poste Italiane S.p.A\\Desktop OLD\\PROGETTI VISUAL STUDIO\\PERSONALE\\FANTACONTRATTI\\node_modules\\pg-pool\\index.js:45:11\n    at processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at connectDatabase (C:\\Users\\39349\\OneDrive - Poste Italiane S.p.A\\Desktop OLD\\PROGETTI VISUAL STUDIO\\PERSONALE\\FANTACONTRATTI\\services\\backend\\user-service\\src\\database\\connection.ts:72:24)\n    at Object.<anonymous> (C:\\Users\\39349\\OneDrive - Poste Italiane S.p.A\\Desktop OLD\\PROGETTI VISUAL STUDIO\\PERSONALE\\FANTACONTRATTI\\services\\backend\\user-service\\src\\__tests__\\integration.database.tdd.test.ts:60:7)","timestamp":"2025-09-25T06:50:07.161Z"}
{"code":"28P01","file":"auth.c","length":104,"level":"error","line":"324","message":"Unable to connect to database: password authentication failed for user \"postgres\"","name":"error","routine":"auth_failed","service":"user-service","severity":"FATAL","stack":"error: password authentication failed for user \"postgres\"\n    at C:\\Users\\39349\\OneDrive - Poste Italiane S.p.A\\Desktop OLD\\PROGETTI VISUAL STUDIO\\PERSONALE\\FANTACONTRATTI\\node_modules\\pg-pool\\index.js:45:11\n    at processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at connectDatabase (C:\\Users\\39349\\OneDrive - Poste Italiane S.p.A\\Desktop OLD\\PROGETTI VISUAL STUDIO\\PERSONALE\\FANTACONTRATTI\\services\\backend\\user-service\\src\\database\\connection.ts:72:24)\n    at Object.<anonymous> (C:\\Users\\39349\\OneDrive - Poste Italiane S.p.A\\Desktop OLD\\PROGETTI VISUAL STUDIO\\PERSONALE\\FANTACONTRATTI\\services\\backend\\user-service\\src\\__tests__\\integration.database.tdd.test.ts:84:7)","timestamp":"2025-09-25T06:50:09.331Z"}
{"code":"ENOTFOUND","errno":-3008,"hostname":"nonexistent-host","level":"error","message":"Unable to connect to database: getaddrinfo ENOTFOUND nonexistent-host","service":"user-service","stack":"Error: getaddrinfo ENOTFOUND nonexistent-host\n    at C:\\Users\\39349\\OneDrive - Poste Italiane S.p.A\\Desktop OLD\\PROGETTI VISUAL STUDIO\\PERSONALE\\FANTACONTRATTI\\node_modules\\pg-pool\\index.js:45:11\n    at processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at connectDatabase (C:\\Users\\39349\\OneDrive - Poste Italiane S.p.A\\Desktop OLD\\PROGETTI VISUAL STUDIO\\PERSONALE\\FANTACONTRATTI\\services\\backend\\user-service\\src\\database\\connection.ts:72:24)\n    at Object.<anonymous> (C:\\Users\\39349\\OneDrive - Poste Italiane S.p.A\\Desktop OLD\\PROGETTI VISUAL STUDIO\\PERSONALE\\FANTACONTRATTI\\services\\backend\\user-service\\src\\__tests__\\integration.database.tdd.test.ts:157:9)","syscall":"getaddrinfo","timestamp":"2025-09-25T06:50:15.917Z"}
{"code":"28P01","file":"auth.c","length":104,"level":"error","line":"324","message":"Unable to connect to database: password authentication failed for user \"postgres\"","name":"error","routine":"auth_failed","service":"user-service","severity":"FATAL","stack":"error: password authentication failed for user \"postgres\"\n    at C:\\Users\\39349\\OneDrive - Poste Italiane S.p.A\\Desktop OLD\\PROGETTI VISUAL STUDIO\\PERSONALE\\FANTACONTRATTI\\node_modules\\pg-pool\\index.js:45:11\n    at processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at connectDatabase (C:\\Users\\39349\\OneDrive - Poste Italiane S.p.A\\Desktop OLD\\PROGETTI VISUAL STUDIO\\PERSONALE\\FANTACONTRATTI\\services\\backend\\user-service\\src\\database\\connection.ts:72:24)\n    at Object.<anonymous> (C:\\Users\\39349\\OneDrive - Poste Italiane S.p.A\\Desktop OLD\\PROGETTI VISUAL STUDIO\\PERSONALE\\FANTACONTRATTI\\services\\backend\\user-service\\src\\__tests__\\integration.database.tdd.test.ts:167:7)","timestamp":"2025-09-25T06:50:18.190Z"}
